app/src/componentLibrary/fieldTextFlex/fieldTextFlex.jsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './fieldTextFlex.scss';
+
+const cx = classNames.bind(styles);
+
+const HEIGHT = 72;
+
+export const FieldTextFlex = ({
+  value,
+  readonly,
+  error,
+  placeholder,
+  disabled,
+  refFunction,
+  onChange,
+  onFocus,
+  onBlur,
+  onKeyUp,
+  touched,
+  className,
+  label,
+  helperText,
+}) => {
+  const resizeHeight = (e) => {
+    e.target.style.height = `${HEIGHT}px`;
+    e.target.style.height = `${e.target.scrollHeight}px`;
+  };
+  return (
+    <>
+      {label && <span className={cx('label', { disabled })}>{label}</span>}
+      <textarea
```

Comment:
Did you discuss with UX that icon in the bottom right corner aren't match to design?